ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml

```
@@ -2343,6 +2403,27 @@ keys:
                     transit:
                       type: bool
                       description: Enable EVPN multicast transit mode.
+            default_route_export:
+              type: list
+              description: Enable default-originate per VRF/address family.
```

Setting primary_key will make the key required and must be unique. This ensures that we don't get the same address family twice and also ensures we always get the address_family set.
```suggestion
            default_route_export:
              type: list
              primary_key: address_family
              description: Enable default-originate per VRF/address family.
```

---

- Requested change performed
As suggested:
- Renamed list `default_route_export` to `default_route_exports`
- Renamed the iterator within the J2 template